suite/tests/client-interface/attach_test.dll.c

```diff
@@ -39,14 +39,17 @@
 
 static thread_id_t injection_tid;
 static bool first_thread = true;
+static bool saw_attach_event = false;
 
 static void
 dr_exit(void)
 {
+    if (!saw_attach_event)
+        dr_fprintf(STDERR, "Error: never saw attach event!\n");
 #ifdef WINDOWS
     dr_fprintf(STDERR, "done\n");
 #else
-    /* The app prints 'done' for us. */
+        /* The app prints 'done' for us. */
```

Comment:
nit: the indentation seems odd. Was it changed by clang-format?

Reply:
Yes, it was clang-format.  The fact that the clang-format-diff check in our test suite here didn't complain means that it agrees: so maybe an old version of clang-format did it the old way and a newer version on my machine and the CI does it this new way?  The new way looks worse...but it doesn't seem worth fighting with clang-format with markers to disable it just for this comment.